### PR TITLE
Guide to partnering with GoCardless

### PR DESCRIPTION
**Pull request by jamesshedden**
Introducing...

partners-guide

A guide to becoming a GoCardless partner, split into two parts - a new `overview` and `faq` sections nested within `/partners`.

Pending final copy!

### CODE REVIEW

**Comment by clippy** [comment]
**Changing existing CSS?**

Make sure you know that your changes to existing CSS aren't having any adverse or unexpected consequences across the app you're working with - this should involve tracking down *any* instance of the class, tag or otherwise that you're changing and checking it out locally to make sure everything looks :ok_hand:.

The way things appear affects how GoCardless is perceived - think more "Tech? Internet? We got this!" and less "Your money is safe with us, probably".

**Adding new CSS?**

If you're adding a new component, make sure your naming of both the filename and classes is consistent and follows our conventions:

Here's what's :+1::

```
/* hero.css */

/* The parent class for your component */
.hero {}

/* Child elements denoted by double undescore */
.hero__image {}

/* Modified versions (which should still depend on their unmodified counterparts)
denoted by double hyphen */
.hero__image--large {}
```

Here's what's :-1::

```
/* page-hero.css */

/* Doesn't match filename */
.hero {}

/* Parent-child relationship suggested by hyphen instead of double underscore */  
.hero-image {}

/* No, just no. */
.hero-image__large {}
```

**Comment by timrogers** [comment]
:pray:

**Comment by jamesshedden** [comment]
@swillis Fancy reviewing this one matey? :100:

File: app/pages/partners/guide/getting-started/partners-guide-getting-started.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        </p>
+
+        <p>
+          You can start developing and testing out our API immediately using a sandbox
+          account <a href='https://manage-sandbox.gocardless.com/signup' target='_blank'>here</a>.
+        </p>
+
+        <p>
+          If you have any questions, contact our developer support team
+          at <a href='mailto:help@gocardless.com'>help@gocardless.com</a>. We also have a
```
Comment (swillis) [inline app/pages/partners/guide/getting-started/partners-guide-getting-started.js]: @jamesshedden Should this use the Support email address saved in `messages`?

File: app/pages/partners/guide/getting-started/partners-guide-getting-started.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          live chat available in your GoCardless dashboard. Our developers are on hand 9am-6pm,
+          Mon-Fri to help with your technical queries.
+        </p>
+
+        <h3>Demo your integration with us and go live</h3>
+
+        <p>
+          Once you have completed your first round of development and testing in our sandbox
+          environment, get in touch with us at <a href='mailto:partnerships@gocardless.com'>partnerships@gocardless.com</a> to
```
Comment (swillis) [inline app/pages/partners/guide/getting-started/partners-guide-getting-started.js]: @jamesshedden Likewise should this use the Partnerships email address saved in `messages`?

File: app/pages/partners/guide/introduction/partners-guide-introduction.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        </p>
+
+        <p>
+          Finally, many customers prefer to pay via Direct Debit, so offering this payment method helps to <b>increase conversion</b> and
+          grow revenue. This is especially the case in Germany and Netherlands, where card penetration is much lower than the UK or US.
+        </p>
+
+        <p>
+          For more information on the benefits of using Direct Debit, read our detailed
+          guides <a href='https://gocardless.com/guides'>here</a>.
```
Comment (swillis) [inline app/pages/partners/guide/introduction/partners-guide-introduction.js]: @jamesshedden External link so needs `target='_blank'`

File: app/pages/partners/overview/partners-overview-page.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+          <p className='u-text-heading-light u-text-m u-color-invert u-padding-Vs'>
+            Our quick guide to becoming a GoCardless partner
+          </p>
+        )}>
+          {this.props.children}
+        </PartnersGuidePage>
+      </Page>
+    );
+  }
+}
```
Comment (swillis) [inline app/pages/partners/overview/partners-overview-page.js]: @jamesshedden Think this file can be removed? :scissors:

**Pull request merged**